src/components/Modal/index.tsx

```diff
@@ -40,8 +47,15 @@ const Modal = ({
         borderRadius="md"
```

Comment:
In line with other comment, I think we can remove this and just use the `base` token declared in the theme file

```suggestion
```